playground/backend/internal/db/entity/common.go

```diff
@@ -21,5 +21,5 @@ type IDMeta struct {
 }
 
 type DatastoreEntity interface {
```

Comment:
I don't get why we need this abstraction: ExampleEntity | SnippetEntity | DatasetSnippetEntity are all structs without any methods